waspc/ChangeLog.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [WIP] 0.12.0
+
+### ⚠️ [Breaking change] Wasp now works with any Node version >= 20.8.1
+So far, Wasp required specific Node version that is compatible with latest LTS Node (lately that was 18).
+
+We relaxed that constraint so it works with any Node version older than the oldest LTS version that Wasp supports.
```

Comment:
You mean any Node version that is newer than the oldest LTS?

Reply:
Thanks, fixed!